music21/stream/__init__.py

```diff
@@ -6247,6 +6249,13 @@ def stripTies(self,
                             retainContainers=True)
             return returnObj  # exit
 
+        if returnObj.hasVoices():
+            for v in returnObj.voices:
+                # already copied if necessary; edit in place
+                v.stripTies(inPlace=True, matchByPitch=matchByPitch,
+                            retainContainers=retainContainers)
+            return returnObj  # exit
+
         # need to just get .notesAndRests, as there may be other objects in the Measure
         # that come before the first Note, such as a SystemLayout object
         f = returnObj.flat
```

Comment:
the line below is where were stripping clefs, etc., when `notes` was returned at the end of the function